addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -43,12 +43,14 @@ def _domain_task_id(self):
         'project.task', 'Task', index='btree_not_null',
         compute='_compute_task_id', store=True, readonly=False,
         domain="[('company_id', '=', company_id), ('project_id.allow_timesheets', '=', True), ('project_id', '=?', project_id)]")
+    parent_task_id = fields.Many2one('project.task', related='task_id.parent_id', store=True)
```

Comment:
I am not sure that parent_task_id makes really sense as when having a situation with sub task of sub task ...
I am unsure have the level just above is really interesting. What would be best is to have the top level task. But this would imply storing the top level on the task (through a stored compute) and storing here the related which seems overkill to me.

I will pushback the PO and let you know 🙂